From 21f6ce6c61ef2a6d87b313c45306e4c6b04be9c2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Thu, 17 Oct 2019 22:02:54 +0200
Subject: [PATCH] Actually install standardese in make install

This was removed in 0911de1312ea731053e66ce87b7a46a0fa9a6e5b but it is
obvious from the log why.
---
 tool/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tool/CMakeLists.txt b/tool/CMakeLists.txt
index fedf237..f188f3e 100644
--- a/tool/CMakeLists.txt
+++ b/tool/CMakeLists.txt
@@ -21,4 +21,4 @@ target_include_directories(standardese_tool PUBLIC ${Boost_INCLUDE_DIR})
 target_link_libraries(standardese_tool PUBLIC ${Boost_LIBRARIES})
 
 # install tool
-#install(TARGETS standardese_tool EXPORT standardese DESTINATION "${tool_dest}")
+install(TARGETS standardese_tool EXPORT standardese DESTINATION "${tool_dest}")
-- 
2.23.0

